release-notes/opensearch-release-notes-2.1.0.md

```diff
@@ -0,0 +1,238 @@
+# OpenSearch and Dashboards 2.1.0 Release Notes
+
+## Release Highlights
```

Comment:
Can you help add these to high lights?
1. dynamic node role feature in OpenSearch core,  https://github.com/opensearch-project/OpenSearch/issues/2877 ?
2. support dedicated ml node in ml-commons plugin, https://github.com/opensearch-project/ml-commons/issues/79

Reply:
3. OpenSearch relevancy features in SQL https://github.com/opensearch-project/sql/issues/182
3.1 https://github.com/opensearch-project/sql/issues/184
3.2 https://github.com/opensearch-project/sql/issues/185
3.3 https://github.com/opensearch-project/sql/issues/187

Reply:
@brijos @elfisher @xinlamzn 
4. Multi-terms aggregation support 
https://github.com/opensearch-project/OpenSearch/issues/1629